### DISCUSSION TITLE
[Bug]: Traces not visible in Langfuse

### DISCUSSION THREAD

**Discussion by qymab** [danny-avila/LibreChat#3022] [Troubleshooting] [answered]
### What happened?

I use LiteLLM as a proxy for all my models, allowing me to utilise Langfuse for observability. However, for some reason, it is not displaying again in Langfuse. To confirm this issue, I employed another frontend, Open-WebUI, with no changes to the configuration on the LiteLLM side. On Open-WebUI, the traces are displayed.

### Steps to Reproduce

1. Create a prompt with all default configurations according to the Librechat documentation.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
if LiteLLM is handling the langfuse integration, I don't see why it would work with OUI and not LibreChat. Please consult with the LiteLLM maintainers.

**Comment by danny-avila** [comment, marked as answer]
Native langfuse integration is something I may explore soon

**Reply by qymab** [reply]
Since you will implement a native solution, I assume I can close the discussion.

**Reply by finnless** [reply]
Where can we track langfuse or any observability integration progress?

**Reply by Odrec** [reply]
Any progress with this?

**Reply by davivc** [reply]
I'm interesting in this integration too.

**Comment by MariusWilsch** [comment]
Any news on this?

**Reply by danny-avila** [reply]
https://github.com/danny-avila/LibreChat/discussions/8012#discussioncomment-13544124

LangSmith is working great: https://www.langchain.com/langsmith

If anyone wants to open a PR including the LangFuse team themselves, I would welcome it. Feel free to open an issue with them: 

https://github.com/langfuse/langfuse
